SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -138,7 +138,6 @@ void USpatialGDKSettings::PostInitProperties()
 	// Check any command line overrides for using QBI, Offloading (after reading the config value):
 	const TCHAR* CommandLine = FCommandLine::Get();
 	CheckCmdLineOverrideBool(CommandLine, TEXT("OverrideHandover"), TEXT("Handover"), bEnableHandover);
-	CheckCmdLineOverrideOptionalBool(CommandLine, TEXT("OverrideMultiWorker"), TEXT("Multi-Worker"), bOverrideMultiWorker);
```

Comment:
I don't think we use this currently, but I believe we have in the past and it does concern me slightly to lose this lever. @cmsmithio do you have any thoughts?

Reply:
This same functionality is now achievable via the `OverrideMultiWorkerSettingsClass` cmd line option

Reply:
Yea as long as we can still force single worker from the command line I think we should be good.